File: src/main/java/ifsp/edu/source/Model/Usuario.java
package ifsp.edu.source.Model;

public class Usuario {
	private long id;
	private String nome;
	private String senha;
	private long idConta;
	private double salario;
	
	public double getSalario() {
		return salario;
	}
	public void setSalario(double salario) {
		this.salario = salario;
	}
	public long getId() {
		return id;
	}
	public void setId(long id) {
		this.id = id;
	}
	public String getNome() {
		return nome;
	}
	public void setNome(String nome) {
		this.nome = nome;
	}
	public String getSenha() {
		return senha;
	}
	public void setSenha(String senha) {
		this.senha = senha;
	}
	public long getIdConta() {
		return idConta;
	}
	public void setIdConta(long idConta) {
		this.idConta = idConta;
	}

}

File: lib/models/usuario.dart
class Usuario {
  final String nome;
  final String senha;
  final int idConta;
  final double salario;

  Usuario({
    required this.nome,
    required this.senha,
    required this.idConta,
    required this.salario,
  });

  factory Usuario.fromJson(Map<String, dynamic> json) {
    return Usuario(
      nome: json['nome'],
      senha: json['senha'],
      idConta: json['id_conta'],
      salario: json['salario'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'nome': nome,
      'senha': senha,
      'id_conta': idConta,
      'salario': salario,
    };
  }
}
